packages/core/package.json

```diff
@@ -29,8 +29,12 @@
     "react": ">=16.3.0"
   },
   "devDependencies": {
+    "@emotion/styled": "^10.0.0-really-unsafe-please-do-not-use.2",
     "@types/react": "16.3.18",
     "dtslint": "^0.3.0",
+    "emotion": "^10.0.0-really-unsafe-please-do-not-use.2",
+    "emotion-server": "^10.0.0-really-unsafe-please-do-not-use.2",
+    "emotion-theming": "^10.0.0-really-unsafe-please-do-not-use.2",
     "html-tag-names": "^1.1.2",
     "svg-tag-names": "^1.1.1"
   },
```

Comment:
Does adding `react` here take costs?
If it's the case, should we also remove them from `@emotion/native` things?